src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -84,7 +84,7 @@ internal ItemManager(RecentItemsManager recentItemsManager)
         // Set the size of pool to 1 because we don't expect UpdateCompletionListAsync to be
         // called concurrently, which essentially makes the pooled list a singleton,
         // but we still use ObjectPool for concurrency handling just to be robust.
-        private static readonly ObjectPool<List<MatchResult>> s_listOfMatchResultPool
+        private static readonly ObjectPool<List<MatchResult<VSCompletionItem>>> s_listOfMatchResultPool
```

Comment:
i guess i'll fine out later.  but why did this become generic?  how is this instantiated in a different fashion?

Reply:
This is generic because VSCompletionItem is an Editor type, but I moved MatchResult down to features to re-use the pattern matching.  So to allow the matchresult to store the corresponding completion item, I made it generic